File: ruby/examples/get_filing_products.rb
require_relative '../request.rb'

# Example of GET /filing-products

WEBSITE_URL	= ENV['WEBSITE_URL']
JURISDICTION 	= ENV['JURISDICTION']

params = {
  jurisdiction: JURISDICTION,
  url: WEBSITE_URL
}

BaseRequestRoute.request(:get, '/filing-products', query_params: params)

File: javascript/examples/get_filing_products.js
const baseRequest = require('../base_request.js').baseRequest

const path = require('path');
process.chdir(path.resolve(__dirname, '../../'));
require('dotenv').config();

// Example of GET /filing-products

const WEBSITE_URL = process.env.WEBSITE_URL;
const JURISDICTION = process.env.JURISDICTION;

params = {
  url: WEBSITE_URL,
  jurisdiction: JURISDICTION
}

token = baseRequest.request.token({ path: '/filing-products' });

baseRequest.request.get({ path: '/filing-products', token: token, queryParams: params });